### PR TITLE
Kong helm chart

### PR DESCRIPTION
**Pull request by shashiranjan84**

### CODE REVIEW

File: charts/kong/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          {{ else }}
+          - name: KONG_CASSANDRA_CONTACT_POINTS
+            value: {{ .Values.kong.database.cassandra.contactPoints }}
+          {{ end }}
+          - name: KONG_CASSANDRA_PORT
+            value: {{ .Values.kong.database.cassandra.port | quote }}
+          - name: KONG_CASSANDRA_KEYSPACE
+            value: {{ .Values.kong.database.cassandra.keyspace}}
+          - name: KONG_CASSANDRA_REPL_FACTOR
+            value: {{ .Values.kong.database.cassandra.replcation | quote }}
```
Comment (stackedsax) [inline charts/kong/templates/deployment.yaml]: should be 'replication'

**Comment by stackedsax** [comment]
Just started testing this.  I noticed that even if I set 'useCassandra' to true, it used Postgres.

Will look at it more tomorrow.

File: charts/kong/templates/deployment.yaml (hunk 1 above)
Comment (shashiranjan84) [reply charts/kong/templates/deployment.yaml]: Good catch, I will fix it.

**Comment by shashiranjan84** [comment]
You would need to update values.yaml file, set 'postgresql.enabled' to false and 'cassandra.enabled' to true.

**Comment by stackedsax** [comment]
Ah, right you are.  

Perhaps adding a comment on the kong.database.useCassandra line so that it's clear that there are a few more settings to set would be useful?  Something like:

`useCassandra: false #Don't forget to set 'postgresql.enabled' and 'cassandra.enabled' appropriately`

**Comment by shashiranjan84** [comment]
Thanks @stackedsax, I know I need to work on doc and some template cleaning, I am thinking add a table detailing about all parameters chart takes. 

regarding `loadBalancerIP`, feel free to send the pr, just make sure its conditional and only added when value is provided.

**Pull request merged**